src/cmd/root.go

```diff
@@ -380,5 +380,5 @@ func validateSubIDFile(path string) (bool, error) {
 		}
 	}
 
-	return false, fmt.Errorf("failed to find an entry for user %s in %p", currentUser.Username, path)
+	return false, fmt.Errorf("failed to find an entry for user %s in %s", currentUser.Username, path)
```

Comment:
Could you please split this hunk out into a separate PR because it's related to the unit tests?

Reply:
Yes, sure. I will remove it from this PR and I will add it on a new one.

Reply:
I've created the [480](https://github.com/containers/toolbox/pull/480) PR  with the minor fix on it.